onnxruntime/core/session/inference_session.cc

```diff
@@ -1578,51 +1590,74 @@ common::Status InferenceSession::Initialize() {
       // now that all the transforms are done, call Resolve on the main graph. this will recurse into the subgraphs.
       ORT_RETURN_IF_ERROR_SESSIONID_(graph.Resolve());
 
-      // Currently only the CUDA EP is considered.
+      // Currently CUDA graph is only considered by CUDA EP and TRT EP.
       // If the CUDA EP is part of the providers list for this session AND
       // The CUDA EP is configured to do a graph capture AND
-      // All the graph nodes have been assigned to the CUDA EP,
+      // All the compute graph nodes have been assigned to the CUDA EP,
```

Comment:
I guess the comment needs to address the TRT EP as well (i.e.) if ALL graph nodes are assigned to the TRT EP....